### PR TITLE
P0 cart wishlist page

### PR DESCRIPTION
**Pull request by dhimanaishwarya**
Cart Page:
image

Wishlist page:
image

### CODE REVIEW

**Review by abhijeetekad — APPROVED** [review]

**Review by Megha-Joshi — APPROVED** [review]
Looks amazing! Good to merge.

File: components/cart.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  object-fit: cover;
+}
+
+.card-product-details {
+  padding: 2rem;
+  width: 20rem;
+}
+
+.old-price {
+  margin: 0.5rem;
```
Comment (Sharmaryan) [inline components/cart.css]: You can use px (upto 10px), if units are less than 1rem.

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Sharmaryan) [inline pages/cart.html:7]: You can also use meta tags for author and description

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+</head>
+
+<body>
+    <!-- header starts  -->
+    <header class="header-wrapper">
+        <a class="active" href="../index.html">
+            <h1>BookzKart</h1>
+        </a>
+        <form action="" class="search-container">
```
Comment (Sharmaryan) [inline pages/cart.html:22]: You can remove action attribute, as it is not needed as of now

**Review by Sharmaryan — APPROVED** [review]
Everything else is great, just suggested some changes.

File: components/cart.css (hunk 1 above)
Comment (dhimanaishwarya) [reply components/cart.css]: updated as per suggestion.

File: pages/cart.html (hunk 2 above)
Comment (dhimanaishwarya) [reply pages/cart.html:7]: updated as per suggestion. Thanks for the review.

**Pull request merged**